config.go

```diff
@@ -0,0 +1,45 @@
+package main
+
+// configuration module for cert-checker
+//
+// Copyright (c) 2022 - Valentin Kuznetsov <vkuznet AT gmail dot com>
+//
+import (
+	"encoding/json"
+	"log"
+	"os"
+)
+
+type Configuration struct {
+	Keytab string `json:"keytab"` // keytab file
+	Cert   string `json:"cert"`   // certificate public file
+	Ckey   string `json:"ckey"`   // certificate private file
+}
+
+// Config represents user provided certs and keytab files
+var Config []Configuration
```

Comment:
Since you have list of Configuration sturcs, the name of the variable should be plural, i.e. `Configs`